packages/app/components/DuplicateIcon/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Pressable } from 'react-native';
 import { MaterialIcons } from '@expo/vector-icons';
-import { useRouter } from 'app/hooks/router';
+import { useRouter } from 'expo-router';
```

Comment:
should be coming from app/hooks. we don't use expo router on all platforms

Reply:
yeah, i got it and have changed it